### PR TITLE
AST-2040 - Fixed Quantity updater ui is breaking

### PR DESCRIPTION
**Pull request by er-roshan**
### Description
Fixed Quantity updater ui is breaking by added margin

### Screenshots
https://d.pr/i/2oXlFg

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/css/unminified/compatibility/woocommerce/woocommerce.css [hunk 1]
```diff
@@ -1262,6 +1262,10 @@ p.demo_store[data-position="bottom"],
   margin: 0 1em 0 0;
 }
 
+.woocommerce div.product form.cart div.quantity.ast-vertical-style-applied + .button.single_add_to_cart_button {
```
Comment (imnavanath) [inline assets/css/unminified/compatibility/woocommerce/woocommerce.css:1265]: Is this heavy selector required? I think we can optimize it, as `.ast-vertical-style-applied` selector is unique and you have applied important over there so by anyways with minimal selectors your CSS will work.

File: assets/css/unminified/compatibility/woocommerce/woocommerce.css [hunk 2]
```diff
@@ -1262,6 +1262,10 @@ p.demo_store[data-position="bottom"],
...
+  margin-left: 1.5em !important;
```
Comment (imnavanath) [inline assets/css/unminified/compatibility/woocommerce/woocommerce.css:1266]: Is there any way if we can avoid important CSS?

**Comment by imnavanath** [comment]
@er-roshan please mention changelog for the same in changelog.txt.

File: assets/css/unminified/compatibility/woocommerce/woocommerce.css (hunk 1 above)
Comment (er-roshan) [reply assets/css/unminified/compatibility/woocommerce/woocommerce.css]: @imnavanath I have removed heavy selector and only used .ast-vertical-style-applied

File: assets/css/unminified/compatibility/woocommerce/woocommerce.css (hunk 2 above)
Comment (er-roshan) [reply assets/css/unminified/compatibility/woocommerce/woocommerce.css]: @imnavanath As some css is directly added to style tag I have used !important. And It is not creating any problem to other section.

**Review by imnavanath — APPROVED** [review]

**Comment by imnavanath** [comment]
Closing this PR as this issuw is already fixed in PR https://github.com/brainstormforce/astra/pull/4525

**Pull request closed without merging**